dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/subcommands/impl/AllTest.java

```diff
@@ -61,7 +60,53 @@ public void testForkRepositoriesFound() throws Exception {
         all.loadDockerfileGithubUtil(dockerfileGitHubUtil);
         all.forkRepositoriesFound(ArrayListMultimap.create(), ArrayListMultimap.create(), contentsWithImage, "image");
 
-        Mockito.verify(dockerfileGitHubUtil, times(1)).closeOutdatedPullRequestAndFork(any());
+        Mockito.verify(dockerfileGitHubUtil, times(3)).closeOutdatedPullRequestAndFork(any());
+    }
+
+    @Test
+    public void testForkRepositoriesFound_unableToforkRepo() throws Exception {
```

Comment:
Please either add a comment on what this is testing. It seems like it would take some time to figure out what went wrong with this test if it started failing.

Reply:
Added comments.